src/src/App.jsx

```diff
@@ -48,6 +53,11 @@ export default function App() {
           <Route path="topics" element={<TopicsPage />} />
           <Route path="capabilities" element={<CapabilitiesPage />} />
           <Route path="ecr" element={<ECRPage />} />
+          <Route path="release-notes" element={<ReleaseNotes />} />
+          <Route path="release-notes/create" element={<ReleaseNotesCreate />} />
```

Comment:
This is different from what we normally do.
I am not against it, as it is delightfully explicit and less based on modals.